### ISSUE TITLE
[Bug] Deleting recipe causes nav bar change

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#3150]
### Description

Deleting a recipe from MyRecipes CollectionView causes the NavBar style to change. It changes in color, and introduces a hamburger menu icon associated with an empty flyout menu.

### Steps to Reproduce

1. Run recipes app
2. Navigate to My Recipes tab
3. Click on a recipe
4. Press the 'Delete' button
5. Notice the changes made to the nav bar

### Expected Behavior

No change to NavBar

### Actual Behavior

Changes in NavBar

### Screenshots

![Screenshot_20211027-125521](https://user-images.githubusercontent.com/21988533/139129952-d5f00632-84be-4d6e-bdb5-70bc54668f80.png)

**Comment by Redth** [comment]
This should be fixed now, it's fairly old and lots of nav changes since, so closing.

**Comment by rachelkang** [comment]
Reopening just until the recipes app accessibility validation is complete! All issues with the a11y/recipes-app label are tracking bugs related to the recipes app, for the accessibility team's reference. The version of the recipes app currently being validated is not up to date with main and still has all these issues.

**Comment by rachelkang** [comment]
future Rachel has come here to say that somehow this is still an issue! :o

**Comment by rachelkang** [comment]
still an issue :(

**Comment by rachelkang** [comment]
Retested with .NET 8 - still an issue

https://user-images.githubusercontent.com/21988533/225160926-1dfbbaf1-75d4-4a4e-9456-4883571dbdd6.mp4
